docs/10-WebServices.md

```diff
@@ -19,21 +19,43 @@ Configure modules in `api.suite.yml`:
 ``` yaml
 class_name: ApiTester
 modules:
-    config:
-		- REST:
-		    url: http://serviceapp/api/v1/
-		    depends: PhpBrowser
+    enabled:
+				- REST:
+				    url: http://serviceapp/api/v1/
+				    depends: PhpBrowser
 ```
```

Comment:
right now: `vendor/bin/codecept generate:suite api` do not generate the previous lines but this:

```
class_name: ApiTester
modules:
    enabled: [\Helper\Api]
```

Should generate command be updated to add the "url"?